### ISSUE TITLE
Update Required? CRASH!

### ISSUE THREAD

**Issue by Wastelander** [niveuseverto/fomm#23]
So, I want to reuse FOMM to install some of the more pesky mods that NMM wholeheartedly refuses to install properly or finds too large a mod to install. However, after a fresh install of the newest FOMM I get the messages at the bottom of this post. There is no crashdump.txt either. This happens when I launch right away and doesn't allow me to change anything. 

What I have done:
-Removed, rebooted, re-installed a fresh copy (no avail) 
-Did my own trouble shooting, watched videos (all unhelpful or outdated), found nothing
-Cried in a corner for several minutes
-Threatened to take its money
-Gave up and searched for a better way to install TTW
-Couldn't find a better way
-Threatened to steal its shoes
-Looked for a way to stop it from updating (couldn't find a way)

So, here is where I sit currently. I cannot play FO:NV because I have a solid black screen because the mod I want to play is only half installed and I cannot get FOMM to do anything other than laugh and twist its mustache. Halp.

FOMM needs to update some of its files. This could take a few minutes, depending on how many mods are installed. 

An error occurred while upgrading your log file. As long as it wasn't too bad, a crash dump will have been saved in BLAH BLAH BLAH\FOMM\crashdump.txt please include the contents of that file if you want to make a bug report.

**Comment by EvilOssie** [comment]
I'm guessing that you have the needed stuff to run TTW to begin with and go from there...(threatened to steal its shoes...lol...)

The game crashes right as it (tries to) load? That's where your solid black screen is?

**Comment by Wastelander** [comment]
Pardon, it's just FOMM not even opening. While yes my game recently broke (I'm being lazy right now to fix it because I want to fix FOMM first) it's really FOMM that isn't working.

The program itself is trying to update and not getting anywhere. It crashes the moment it's opened and says it has to update. Then doesn't make a crashdump. (Sorry in writing this on my phone so I hope you can understand, if too vague I'll elaborate when I get an actual keyboard in front of me.)

**Comment by EvilOssie** [comment]
Try this...

 - check your fomod directory for a TTW fomod; if it's there, delete it
 - uninstall FOMM as completely as you can, then reinstall in a different location from the previous one. That, hopefully, will 'reset' the failed update.

And just in case...what broke your game?

**Comment by Wastelander** [comment]
No worries I finally got FOMM up and running after a bit of tweaking. My game isn't broken exactly, it's just I disabled a number of mods in bulk that need to be running due to how integrated they now are with my game and such,

**Comment by niveuseverto** [comment]
Please, let me info about mods, that made FOMM unstable like that, I'll check them out.

**Comment by niveuseverto** [comment]
Released here and on NexusMods latest version. If problem persists, feel free to reopen this issue.

**Comment by onimale** [comment]
Right I'm having the issue..I get a crash dump however I have tried everything I can imagine and nothing works I posted on nexusmods as well sorry if I seem annoying but I can't play FO:NV till I get this fixed Dump follows.

Sunday, May 17, 2015 - 9:28:38 AM
Fomm 0.13.21
OS version: Microsoft Windows NT 6.1.7601 Service Pack 1

System.Xml.XmlException: Data at the root level is invalid. Line 1, position 1.
   at Fomm.BackgroundWorkerProgressDialog.System.IDisposable.Dispose()
   at Fomm.InstallLogUpgraders.Upgrader.PerformUpgrade()
   at Fomm.InstallLogUpgraders.InstallLogUpgrader.UpgradeInstallLog()
   at Fomm.Program.Main(String[] args)
